dataflow/src/main/java/org/checkerframework/dataflow/reachingdefinitions/ReachingDefinitionsNode.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.dataflow.reachingdefinitions;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.cfg.node.AssignmentNode;
+
+/**
+ * A ReachingDefinitionsNode contains a CFG node, which can only be a AssignmentNode. It is used to
+ * represent the estimate of reaching definitions at certain CFG block during dataflow analysis. We
```

Comment:
The plural in `reaching definitions` is odd, as there is only a single `AssignmentNode` field. So should this be singular, in the description and class name?

Reply:
Yes, thanks!